Template10 (Library)/Services/NavigationService/NavigationService.cs

```diff
@@ -291,7 +295,8 @@ public void Navigate<T>(T key, object parameter = null, NavigationTransitionInfo
         }
 
         public event TypedEventHandler<Type> AfterRestoreSavedNavigation;
-        public async Task<bool> RestoreSavedNavigationAsync()
+
+        public async Task<bool> LoadAsync()
```

Comment:
Sorry to be picky about this, but we can't change the external signature of the class. There may very well be developers using RestoreSavedNavigationAsync and we can't rename it. Instead, mark it with [Obsolete] and create a duplicate called LoadAsync(). Make sense?